cli/src/action/playlist.rs

```diff
@@ -203,13 +203,13 @@ impl Action for SubmitPlaylistAction {
             .ok_or_else(|| CliError::ActionError("'key' is required".into()))?;
         let (auth, _) = create_cylinder_jwt_auth_signer_key(key_path)?;
 
-        let rate: u32 = args
+        let rate: f32 = args
             .value_of("rate")
             .unwrap_or(DEFAULT_RATE)
             .parse()
             .map_err(|_| CliError::ActionError("Unable to parse provided rate".into()))?;
 
-        if rate == 0 {
+        if rate == 0.0 {
```

Comment:
Previously this was an unsigned comparison; now it should also check for values less than zero.

Reply:
changed to <=